CompositeUi/src/views/component/Search0.js

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
Search0.js ??? WTF Search.js file is already exists so, you can remove safely this duplicate file